en/middleware/mavlink.md

```diff
@@ -518,4 +518,4 @@ However if you use your own dialect then it should include ArduPilotMega.xml (or
 
 ### Updating MAVSDK
 
-See the MAVSDK docs for information about how to work with [MAVLink headers and dialects](https://mavsdk.mavlink.io/main/en/cpp/guide/build.html#mavlink-headers-and-dialects).
+See the MAVSDK docs for information about how to work with [MAVLink headers and dialects](https://mavsdk.mavlink.io/main/en/cpp/guide/build.html).
```

Comment:
@julianoes This pointed to "MAVLink headers and dialects". That heading is no longer present - is there a "right place" for this?

Reply:
It's not really documented at the moment.
https://github.com/search?q=repo%3Amavlink%2FMAVSDK%20MAVLINK_DIALECT&type=code

I plan on making adding dialects easier in the future, so it's not worth adding docs around it now though.

Reply:
Thanks. I'm just going to leave it as it is then!